doc/en/user/source/security/urlchecks.rst

```diff
@@ -3,65 +3,110 @@
+
+.. _security_urlchecks_edit:
+
+Editing a URL Check
+-------------------
 
-- **Name**: a name for the check, used to identify it in the list.
-- **Description**: a description of the check, for later reference.
-- **Regular expression**: a regular expression to match the URL to check.
-- **Enabled**: a check box to enable or disable the check.
+Regular Expression URL checks can be configured, with the following parameters for each check:
 
-.. figure:: images/regex.png
+.. list-table::
+   :widths: 30 70 
+   :header-rows: 1
 
-   The regular expression based check
+   * - Field
+     - Description
+   * - Name
+     - Name for the check, used to identify it in the list.
+   * - Description
+     - Description of the check, for later reference.
+   * - Regular Expression
+     - A regular expression used to match allowed URLs
+   * - Enabled
+     - Check box to enable or disable the check
 
-The most common URL patter type, allowing matches prefixed by a given host name, follows the following pattern:
+The most common URL pattern type, allowing matches prefixed by a given host name, follows the following pattern:
 
-``^https?://(www\.)?example\.com/.*$`` 
+``^https?://(www\.)?example\.com/.*$``
 
-Numerous web site can help building a valid Java regular expression patter, interpret and explain it,
-as well as test it. For example:
+Example allowing WMS ``REMOTE_OWS`` data access to an external WFS service:
 
-* https://regex101.com/ (enabled the Java 8 flavor)
-* https://www.freeformatter.com/java-regex-tester.html 
+``^https://safeWFS/geoserver/ows/.*SERVICE=WFS.*$``
 
+.. figure:: images/urlchecks-edit.png
+   
+   Configure Regular Expression URL check
+    
+.. note::
+
+   Web sites are available to can help define a valid Java regular expression pattern. These tools can be used to interpret, explain and test regular expressions. For example:
+
+   * https://regex101.com/ (enabled the Java 8 flavor)
+   * https://www.freeformatter.com/java-regex-tester.html 
 
 Testing URL checks
 ------------------
 
-The bottomg of the URL checks page contains a form to test the URL checks. The form allows to enter
-a given URL, and will perform the checks on it, reporting the results.
+The :guilabel:`Test URL Checks with external URL` form allows a URL to be checked, reporting if access is allowed or disallowed.
+
+Test URL Checks form:
+
+.. list-table::
+   :widths: 30 70 
+   :header-rows: 1
+
+   * - Field
+     - Description
+   * - URL to check 
+     - Supply URL of external resource to check if access is allowed
+
+Press :guilabel:`Test URL` button to perform the checks. If at least one URL Check matches the URL, it will be allowed and the test will indicate the URL Check permitting access. Otherwise it will be rejected and the test will indicate that no URL Check matched.
```

Comment:
```suggestion
Press the :guilabel:`Test URL` button to perform the checks. If at least one URL Check matches the URL, it will be allowed and the test will indicate the URL Check permitting access. Otherwise it will be rejected and the test will indicate that no URL Check matched.
```